target/linux/ath79/dts/qca9531_glinet_gl-ar300m-lite.dts

```diff
@@ -7,7 +7,6 @@
 	model = "GL.iNet GL-AR300M-Lite";
 };
 
-/delete-node/ &reg_usb_vbus;
```

Comment:
Why don't you delete the `gpio_usb_power` too?

Reply:
We need to export GPIO to user space beforehand so that users can control USB power through it.
Of course, you can also ask the user to export it manually at the time of use, but glinet's firmware will be directly based on openwrt, and our API will use the export node directly.